R/SS_html.R

```diff
@@ -351,11 +351,16 @@ SS_html <- function(replist = NULL,
         file = htmlfile, append = TRUE
       )
       for (i in 1:nrow(plotinfo)) {
-        # default alternative text is caption up to any line break <br>
+        # default alternative text is caption up to any line break
```

Comment:
@iantaylor-NOAA having the caption repeated as alternative text makes the individual using screen readers have to listen to the caption twice and is thus seen as less ideal than having no alternative text.

Reply:
Thanks @kellijohnson-NOAA. This change makes things slightly less bad than the status-quo by reducing some multi-sentence alt-text to single sentence, but it's easy to change it to "" instead. Is that what you would suggest?

Reply:
Thanks for the feedback on this, @kellijohnson-NOAA. The alt-text is now "" throughout. I will transfer to #411 the suggestion about populating alt-text with reference to the sections in the report file.